.github/ISSUE_TEMPLATE.md

```diff
@@ -1,27 +1,4 @@
-<!--
-THIS IS NOT A HELP FORUM.
-If you are experiencing problems with setting up Jest, please make sure to visit our Help page:
-https://facebook.github.io/jest/help.html
--->
+## 👉 [PLEASE CHOOSE AN ISSUE TEMPLATE HERE](https://github.com/facebook/jest/issues/new/choose) 👈
```

Comment:
I updated the default to redirect to chose one of the templates, but is this too aggressive?

Reply:
I like it